cmd/start.go

```diff
@@ -73,6 +74,16 @@ func Start(repoFlagName string, tp node.Type) *cobra.Command {
 				opts = append(opts, node.WithConfig(cfg))
 			}
 
+			mutualPeersStr := cmd.Flag(mutualPeersFlag.Name).Value.String()
+			if mutualPeersStr != "" {
+				mutualPeers := strings.Split(mutualPeersStr, ",")
+				if len(mutualPeers) == 0 {
+					return fmt.Errorf("%s flag is passed but no address were given", mutualPeersFlag.Name)
+				}
+
+				opts = append(opts, node.WithMutualPeers(mutualPeers))
+			}
+
```

Comment:
Q: Should this be refactored into a common function?

Reply:
Yeah, eventually.

Reply:
can u do in this PR? just make it a common flag the way the other common flags were declared.

Reply:
Otherwise we'd never get to it 😅

Reply:
> can u do in this PR?
Yesss